components/private/Math/src/MDCMath.h

```diff
@@ -30,6 +30,10 @@ static inline CGFloat MDCFabs(CGFloat value) {
 #endif
 }
 
+static inline CGFloat MDCDegreesToRadians(CGFloat degrees) {
+  return degrees * M_PI / 180.f;
```

Comment:
Could you make it double if cgfloat is double and float if not?

Reply:
Same comment from #2016 
With the explicit 180.f there is a cast if CGFLOAT_IS_DOUBLE.

#if CGFLOAT_IS_DOUBLE
return degrees * M_PI / 180.0;
#else
return degrees * M_PI / 180.f;
#endif